.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/.github/workflows"
+    schedule:
+      interval: "daily"
```

I'd prefer a weekly interval (even monthly would be OK).

```suggestion
      interval: "weekly"
```

---

Weekly works as well, I just prefer to get it as soon as possible and it won't spam me anyways because I almost always only use major version specifiers in github actions but if you use minor or even patch versions that may make more sense as a longer interval.

---

OK, we can stick to daily schedules then.